proposals/wac-ucr/index.bs

```diff
@@ -802,6 +802,20 @@ For example, if the data Carol and Oscar saw in the resume was
 background, she wouldn't want them to know that they were only seeing
 a filtered view.
 
+### High Confidentiality Scenario ### {#uc-confidential} 
+
+A service with very high security and confidentiality requirements must publish 
+resources without the URL for each resource leaking information about it's content.
+This service publishes all these resources at a domain with each document named
+by a secure hashing under one root container. Some of these resources at those
```

Comment:
Why hashing and not just a random uuid?

Reply:
I just meant the path names to be opaque. Perhaps UUIDs are a better idea there, but they don't need to be globally unique.

Reply:
I agree that hashing may only bring unintended confusion. UUID seems like a safer alternative for this use case.

Reply:
I also agree that the concept of a hash brings with it the question "hash of what?", where an default answer tends to be "of the content, of course", which is not the intention at all. A [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) on the other hand stands for a Univerally Unique Identifier, which is too strong for the use case. Is there a word that captures something in between?
Looking for "Locally Unique Identifiers" I found quite a few questions  concerning those on Stack Exchange and even a longer [blog post from 2010](http://wisdomofganesh.blogspot.com/2010/06/case-for-locally-unique-ids-luids.html).

Reply:
https://www.w3.org/TR/capability-urls/#capability-url-design

> Good unique URLs include an unguessable unique identifier such as a UUID. 

------

Even though we don't talk about capability urls, i think this suggestion for unguessable unique identifiers applies here as well.

Reply:
👍 to UUID